lib/resources/packages.rb

```diff
@@ -23,12 +23,17 @@ class Packages < Inspec.resource(1)
     "
 
     def initialize(pattern)
-      @command = packages_command
-
-      return skip_resource "The packages resource is not yet supported on OS #{inspec.os.name}" unless @command
+      os = inspec.os
+      if os.debian?
+        @pkgs = Debs.new(inspec)
+      elsif %w{redhat suse amazon fedora}.include?(os[:family])
```

Comment:
I used the same checks as the existing `package` resource:
```ruby
      elsif %w{redhat suse amazon fedora}.include?(os[:family])
        @pkgman = Rpm.new(inspec)
```

Noticed that `os.redhat?` is not matching Fedora:
```ruby
inspec> os.redhat?
=> false
inspec> os.params
=> {:name=>"fedora", :family=>"fedora", :release=>"25", :arch=>"x86_64"}
```

If we go with `os.redhat?`, will need something like:
```
lib/resources/grub_conf.rb
35:    if os.redhat? || os[:name] == 'fedora'
```